CHANGELOG.md

```diff
@@ -3,6 +3,59 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.15.0
+
+_Jul 28, 2022_
+
+We'd like to offer a big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Aggregation (#5592) @flaviendelangle
+
+  Premium users can now aggregate data in the grid.
+  You can now use aggregation functions, and with a couple of clicks, extract information like sum, average, count, and others
+
+  ![modify aggregation](https://user-images.githubusercontent.com/45398769/181478185-142f87b0-c8de-4585-b381-cf4fdc62d99f.gif)
+
+  To enable the feature, add `experimentalFeatures={{ aggregation: true }}`.
+  Aggregation function can be customized, and interact with row aggregation.
```

Comment:
```suggestion
  Aggregation functions are customizable and they combine well with row grouping (inline aggregations).
```

Reply:
```suggestion
  Aggregation functions are customizable and they combine well with row grouping.
```

I do not add the inline aggregation because it's kind of internal vocabulary (nothing comes to my mind when seeing this expression)